articles/azure-resource-manager/bicep/bicep-functions-files.md

```diff
@@ -21,11 +21,11 @@ Namespace: [sys](bicep-functions.md#namespaces-for-functions).
 
 | Parameter | Required | Type | Description |
 |:--- |:--- |:--- |:--- |
-| filePath | Yes | string | The path to the file to load. The path is relative to the deployed Bicep file. |
+| filePath | Yes | string | The path to the file to load. The path is relative to the deployed Bicep file and it should be a compile-time constant (cannot use variables)  |
 
 ### Remarks
 
-Use this function when you have binary content you would like to include in deployment. Rather than manually encoding the file to a base64 string and adding it to your Bicep file, load the file with this function. The file is loaded when the Bicep file is compiled to a JSON template. During deployment, the JSON template contains the contents of the file as a hard-coded string.
+Use this function when you have binary content you would like to include in deployment. Rather than manually encoding the file to a base64 string and adding it to your Bicep file, load the file with this function. The file is loaded when the Bicep file is compiled to a JSON template (Hence variables cannot be used in filepath, as they are not resolved at this stage). During deployment, the JSON template contains the contents of the file as a hard-coded string.
```

Comment:
```suggestion
Use this function when you have binary content you would like to include in deployment. Rather than manually encoding the file to a base64 string and adding it to your Bicep file, load the file with this function. The file is loaded when the Bicep file is compiled to a JSON template. Hence variables cannot be used in filePath as they are not resolved at this stage. During deployment, the JSON template contains the contents of the file as a hard-coded string.
```